sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -133,52 +139,27 @@ export class InstitutionLocationsController extends BaseController {
    * @param userToken
    * @returns All the institution locations for the given institution.
    */
+  @ApiOkResponse({
+    description: "Institution locations found.",
+  })
+  @ApiNotFoundResponse({
+    description: "Institution locations not found.",
+  })
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @IsInstitutionAdmin()
   @Get()
   async getAllInstitutionLocations(
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<InstitutionLocationsDetailsDto[]> {
     // get all institution locations.
-    const InstitutionLocationData =
-      await this.locationService.getAllInstitutionLocations(
+    const DesignationAgreementLocationData =
```

Comment:
Local variables should start with a lower case, please see the wiki
https://github.com/bcgov/SIMS/wiki/Development-Standards-and-Decisions#sample-class-example